src/smc_mallows_new_item_rank_alpha_fixed.cpp

```diff
@@ -164,28 +169,32 @@ Rcpp::List smc_mallows_new_item_rank_alpha_fixed(
   }
 
   /* normalise weights ------------------------------------ */
-  double maxw = arma::max(log_inc_wgt);
-  arma::vec w = arma::exp(log_inc_wgt - maxw);
-  arma::vec norm_wgt = w / arma::sum(w);
+  double maxw = max(log_inc_wgt);
+  vec w = exp(log_inc_wgt - maxw);
+  vec norm_wgt = w / sum(w);
+
+  /* store ESS = sum(w)^2/sum(w^2) */
+  ESS_vec(tt) = (sum(norm_wgt) * sum(norm_wgt)) / sum(norm_wgt % norm_wgt);
```

Comment:
There's no `tt` iteration here, so the left side needs to be adjusted.

Reply:
@anjastein, it looks like if you replicate the merged changes from PR #192 here, [particularly the changes on smc_mallows_new_item_rank()](https://github.com/ocbe-uio/BayesMallows/pull/192/files#diff-ef31c5f7bbfbf73a256af7db1019ad9db78041cadf42831c34e033bb2817c870), this will get resolved. Remember to change unit test expectations accordingly, and run `devtools::checks()` on your local machine before pushing any changes.